js/src/lib/components/InferenceWidget/shared/WidgetTextarea/WidgetTextarea.svelte

```diff
@@ -82,13 +83,11 @@
 	<svelte:fragment slot="after">
 		<!-- `whitespace-pre-wrap inline-block` are needed to get correct newlines from `el.textContent` on Chrome -->
 		<span
-			class="{isLoading ? 'pointer-events-none' : ''} {label
-				? 'mt-1.5'
-				: ''} block w-full resize-y overflow-auto py-2 px-3 {size === 'small'
+			class="{label ? 'mt-1.5' : ''} block w-full resize-y overflow-auto py-2 px-3 {size === 'small'
 				? 'min-h-[42px]'
 				: 'min-h-[144px]'} inline-block max-h-[500px] whitespace-pre-wrap rounded-lg border border-gray-200 shadow-inner outline-none focus:shadow-inner focus:ring focus:ring-blue-200 dark:bg-gray-925"
 			role="textbox"
-			contenteditable
+			contenteditable={!isLoading && !isDisabled}
```

Comment:
i think this shouldn't take focus if no editable

More generally (unrelated to this PR), I think it shouldn't take focus nor be editable while tex generation is happening